lib/consyncful/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Consyncful
-  VERSION = '0.4.0'
+  VERSION = '0.4.1'
```

Comment:
(discussed in person)

```suggestion
  VERSION = '0.5.0'
```